mlir/include/mlir/Target/LLVMIR/LLVMTranslationInterface.h

```diff
@@ -90,6 +102,20 @@ class LLVMTranslationInterface
     }
     return success();
   }
+
+  /// Acts on the given function operation using the interface implemented by
+  /// the dialect of one of the function parameter attributes.
+  virtual LogicalResult
+  convertParameterAttr(LLVM::LLVMFuncOp function, int argIdx,
+                       NamedAttribute attribute,
+                       LLVM::ModuleTranslation &moduleTranslation) const {
+    if (const LLVMTranslationDialectInterface *iface =
+            getInterfaceFor(attribute.getNameDialect())) {
+      return iface->convertParameterAttr(function, argIdx, attribute,
+                                         moduleTranslation);
+    }
+    return success();
```

Comment:
I'm not sure the default here should be success. This would be dropping any attribute from a dialect without interface on the floor without warning.

Reply:
Will modify it to return failure. I followed what `amendOperation` was doing for op attributes.

Reply:
Returning failure causes several test failures. There are dialect attributes like 'fir.bindc_name' which doesn't require any handling here. I updated it to emit a warning.

Reply:
(unresolving, @ftynse should take another look here and acknowledge the solution explicitly)